atd-etl/cris_import/cris_import.py

```diff
@@ -592,6 +719,10 @@ def align_records(map_state):
                 # To decide to UPDATE, we need to find a matching target record in the output table.
                 # This function returns that record as a token of existence or false if none is available
                 if util.fetch_target_record(pg, output_map, table, record_key_sql):
+                    
+                    # Cause the CR3s to be re-downloaded next time that ETL is run
+                    util.invalidate_cr3(pg, source["crash_id"])
```

Comment:
If we're in this execution branch, we're in the "update" branch, so invalidate the CR3.